content/en/synthetics/troubleshooting/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Synthetics Troubleshooting
+kind: documentation
+description: Troubleshoot common Synthetics isses.
+beta: true
+further_reading:
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/browser_tests"
+  tag: "Documentation"
+  text: "Configure a Browser Test"
+- link: "synthetics/api_tests"
+  tag: "Documentation"
+  text: "Configure an API Test"
+---
+
+If you experience issues setting up or configuring Datadog Synthetics, use this information to begin troubleshooting. If you continue to have trouble, [contact our awesome support team][1].
+
+## Failed requests on a working endpoint
+
+You can sometimes get `5xx` or `4xx` when setting an API or Browser test on your endpoints. This happens most when hitting `Test URL` to perform a fast test, and then the error shows up in your regular test results. However, you know there's not actually an issue, because when you visit your website in a browser (or cURL it), you're getting `2xx`. 
+
+This happens because cURL automatically sets a `user-agent` as a request header, and this also automatically happens when you visit the website in your browser. Datadog API tests and Browser tests don't automatically set a `user-agent`. This can be a problem because some websites ban requests that don't have a `user-agent` set, which causes Datadog API tests and Browser tests to return a `5xx` or `4xx`.
+
+The solution to this problem is to manually set a `user-agent` in your API tests and Browser. Set the `user-agent` in your API or Browser test under **Make a request** > **Advanced Options** > **Header** > **Request Header**. Set the **Name** to `user-agent` and set the **Value** to the `user-agent` value.
```

Comment:
That's not needed for browser tests in the vast majority of cases. We should not need to train users to do so if they don't need it. Overall this whole article as written almost only applies to API tests (not browser nor ssl tests).